Source/WebKit/UIProcess/RemoteLayerTree/mac/ScrollingTreeFrameScrollingNodeRemoteMac.cpp

```diff
@@ -61,11 +61,14 @@ void ScrollingTreeFrameScrollingNodeRemoteMac::repositionRelatedLayers()
     m_delegate->updateScrollbarLayers();
 }
 
+void ScrollingTreeFrameScrollingNodeRemoteMac::handleWheelEventPhase(const PlatformWheelEventPhase phase)
+{
+    m_delegate->handleWheelEventPhase(phase);
+}
+
 WheelEventHandlingResult ScrollingTreeFrameScrollingNodeRemoteMac::handleWheelEvent(const PlatformWheelEvent& wheelEvent, EventTargeting eventTargeting)
 {
-    auto result = ScrollingTreeFrameScrollingNodeMac::handleWheelEvent(wheelEvent, eventTargeting);
-    m_delegate->handleWheelEventForScrollbars(wheelEvent);
-    return result;
+    return ScrollingTreeFrameScrollingNodeMac::handleWheelEvent(wheelEvent, eventTargeting);
```

Comment:
Can you just remove this override implementation?

Reply:
Fixed.